kratos/processes/sub_model_part_skin_detection_process.h

```diff
@@ -28,49 +28,66 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
-/// Create a SubModelPart covering a part of the outside skin of the computation domain where a condition is met.
-/** For example, create the outer skin for the part of the domain belonging to a given SubModelPart.
-*/
+/**
+ * @class SubModelPartSkinDetectionProcess
+ * @brief Create a SubModelPart covering a part of the outside skin of the computation domain where a condition is met.
+ * @details For example, create the outer skin for the part of the domain belonging to a given SubModelPart.
+ * @author Jordi Cotela
```

Comment:
I think this is @rubenzorrilla

Reply:
let's wait @rubenzorrilla , I copied the header of the file